src/AspNetCore/Inc/forwardinghandler.h

```diff
@@ -70,6 +70,9 @@ class FORWARDING_HANDLER
 
     REQUEST_NOTIFICATION_STATUS
     OnAsyncCompletion(
+        IHttpContext *          pHttpContext,
```

Comment:
I don't think you need those input parameters. forwardinghandler already has m_pW3Context wrapping the request